### ISSUE TITLE
Illegal alt attribute for other videos

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-website#3242]
This is a follow-on from issue https://github.com/corona-warn-app/cwa-website/issues/3227 which covered a problem with the [Explanatory video](https://www.coronawarn.app/en/faq/results/#in_a_nutshell) in the FAQs.

This issue covers the same problem, but in other places on https://www.coronawarn.app.

## Where to find the issue

- https://www.coronawarn.app/en/science/2021-06-15-science-blog-1/
"What is the purpose of evaluating the CWA and what aspects play a role in the evaluation?"

- https://www.coronawarn.app/de/science/2021-06-15-science-blog-1/
"Über die Wirksamkeit und den Nutzen der Corona-Warn-App"

- https://www.coronawarn.app/de/sign-language/ "Gebärdensprache"

## Describe the issue

`<video>` tags on the above pages contain the attribute `alt=`. The `<video>` tag is not allowed to have an `alt` attribute so this is incorrect HTML code.

An activated screen reader does not read out any description of the video contents.

### Validator

Run https://validator.w3.org on 

- https://www.coronawarn.app/en/science/2021-06-15-science-blog-1/ (3 video errors)
- https://www.coronawarn.app/de/science/2021-06-15-science-blog-1/ (2 video errors)
- https://www.coronawarn.app/de/sign-language/ (1 video error)
  
and note the video related error message as follows:

- "Attribute alt not allowed on element video at this point."

### Screen reader

In Windows 10 / 11

1. Open corresponding web page in Google Chrome
2. Enable Narrator: Windows key + Ctrl + Enter
3. Navigate through document with numerical heading keys and tab
4. Move to video
5. Note that it reads (only) "Group"

It does **not** read the alt text.

## Suggestion

Convert the `alt` attribute from the `<video>` content tab to an `aria-label` attribute.

## References

- [HTML `<video>` Tag](https://www.w3schools.com/tags/tag_video.asp)
- [Accessible Publishing Knowledge Base - Video examples](http://kb.daisy.org/publishing/docs/html/video.html#ex)

**Comment by MikeMcC399** [comment]
- I will wait for the outcome of PR https://github.com/corona-warn-app/cwa-website/pull/3228 before starting any work to resolve this one.

**Comment by brianebeling** [comment]
@MikeMcC399 Hey, just wanted to let you know that the aria-label PR is merged, with the intention to revisit the remaining idea of improving the accessibility by using clever wording. Sadly I do not have a lot of experience with narrators, but I'm excited to revisit this when the current backlog is a bit thinner.

**Comment by MikeMcC399** [comment]
@brianebeling 

I will submit a PR to make the other video links valid HTML. That would make all the video links equivalent.

You could then pick up your idea of enhancing the screen reader experience when you have time in a separate PR.
